var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -13,12 +13,13 @@ class Julia(Package):
     """The Julia Language: A fresh approach to technical computing"""
 
     homepage = "http://julialang.org"
-    url      = "https://github.com/JuliaLang/julia/releases/download/v0.4.3/julia-0.4.3-full.tar.gz"
+    url      = "https://github.com/JuliaLang/julia/releases/download/v1.6.2/julia-1.6.2-full.tar.gz"
```

Comment:
Unless the formatting of the URL changes, the older version does not need to be replaced.
```suggestion
```

Reply:
It's not strictly necessary, but also it doesn't hurt to update an ancient version number...